plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -42,9 +48,9 @@ export class AwsOrganizationCloudAccountProcessor implements CatalogProcessor {
   static fromConfig(
     config: Config,
     options: {
-      logger: Logger;
+      logger?: Logger;
```

Comment:
Was this change made for some particular reason?

Reply:
I don't think so after trying to dig through it a little bit